lib/zuora/objects/product_rate_plan_charge.rb

```diff
@@ -16,7 +16,7 @@ class ProductRatePlanCharge < Base
     INCLUDED_UNITS              = ['Tiered Pricing','Tiered with Overage Pricing','Volume Pricing']
     OVERAGE_CALCULATION_OPTIONS = %w(EndOfSmoothingPeriod PerBillingPeriod)
     OVERAGE_UNUSED_UNITS_CREDIT_OPTIONS = %w(NoCredit CreditBySpecificRate)
-    PRICE_INCREASE_OPTIONS = %w(FromTenantPercentageValue SpecificPercentageValue)
+    # PRICE_INCREASE_OPTIONS = %w(FromTenantPercentageValue SpecificPercentageValue)
```

Comment:
Why is this commented out?

Reply:
Resolved here [04f281f](https://github.com/Sage/zuora/pull/18/commits/04f281f9a0a0b07961e1b1f4ebd5465beb8c27bc)